dpcontracts.py

```diff
@@ -546,6 +552,124 @@ def inner(*args, **kwargs):
         return inner
     return func
 
+def pytypes():
+    """
```

Comment:
While having this here is probably the more correct way, for the sake of consistency could we move it into the module's docstring?